extensions/send-to-readwise/src/index.tsx

```diff
@@ -0,0 +1,137 @@
+import { ActionPanel, Form, showToast, Toast, Action, getPreferenceValues } from "@raycast/api";
+import { useForm, FormValidation } from "@raycast/utils";
+import fetch from "node-fetch";
+
+const READWISE_API_URL = "https://readwise.io/api/v2/highlights/";
+
+interface Preferences {
+  apiKey: string;
+}
+
+interface ReadwiseFormValues {
+  content: string;
+  reference: string;
+  note: string;
+}
+
+interface Highlight {
+  title: string;
+  text: string;
+  url?: string;
+  note?: string;
+}
+
+export default function Command() {
+  const { handleSubmit, itemProps, values } = useForm<ReadwiseFormValues>({
+    onSubmit: async (values) => {
+      const preferences: Preferences = getPreferenceValues();
+      const apiKey = preferences.apiKey;
+      function formatReferences(references: string): string {
+        if (!references) {
+          return "";
+        }
+        const refs = references
+          .split(/,|，|\n/)
+          .map((ref) => ref.trim())
+          .filter(Boolean);
+        let formattedRefs = "\n**References**\n";
+        refs.forEach((ref, index) => {
+          formattedRefs += `- [[${index + 1}] ](${ref})`;
+          if (index !== refs.length - 1) {
+            formattedRefs += "\n";
+          }
+        });
+        return formattedRefs;
+      }
+      const references = formatReferences(values.reference);
+      let content = values.content;
+      if (references) {
+        content += `\n${references}`;
+      }
+
+      try {
+        const url = values.reference;
+        const note = values.note;
+        console.log("Sending request with URL:", url);
+        console.log("Sending request with Note:", note);
+        const today = new Date().toISOString().slice(0, 10);
+        const highlights: Highlight[] = [
+          {
+            title: today,
+            text: content,
+            note: note,
+          },
+        ];
+
+        const response = await fetch(READWISE_API_URL, {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+            Authorization: `Token ${apiKey}`,
+          },
+          body: JSON.stringify({ highlights }),
```

Comment:
Please, add support for more supported parameters: 
<img width="805" alt="image" src="https://github.com/raycast/extensions/assets/1043668/c49782ca-4e8a-4e5b-ab66-ae54d80ae52a">